### PR TITLE
WHO Protect Yourself Page

### PR DESCRIPTION
**Pull request by myiremark**

### CODE REVIEW

**Comment by myiremark** [comment]
00_wash_hands
01_cover
02_closed_bin
03_wash_hands_often
04_avoid_close_contact
05_seek_medical_care

File: client/ionic/src/App.tsx [hunk 1]
```diff
@@ -32,6 +33,11 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
         <Route path="/menu" component={Menu} exact={true} />
+        <Route
+          path="/protect_yourself"
```
Comment (hspinks) [inline client/ionic/src/App.tsx]: recognizing that this is totally a personal preference thing, but I prefer `kebab-case` to `snake_case` in URL paths. open to discussion though

Comment (myiremark) [reply client/ionic/src/App.tsx]: i'd say it matters.  WHO uses kebab-case on their website.  I don't have a preference, just python baked into my brain

Comment (advayDev1) [reply client/ionic/src/App.tsx]: these paths can't be seen by the user (at least if we're not a true webapp) so meh. but  i like kebabs too (and kebab case ;)

Comment (myiremark) [reply client/ionic/src/App.tsx]: intention is to get scaffolds in for all pages.  closing this pr, 

reopened as https://github.com/WorldHealthOrganization/app/pull/141

for branch name, also, switching to dynamic content.

Comment (myiremark) [reply client/ionic/src/App.tsx]: will also update link in slack

**Pull request closed without merging**